packages/hoppscotch-app/components/app/PaneLayout.vue

```diff
@@ -56,5 +64,62 @@ const SIDEBAR = useSetting("SIDEBAR")
 
 const slots = useSlots()
 
+const props = defineProps({
+  layoutId: {
+    type: String,
+    default: null,
+  },
+})
+
+type PaneEvent = {
+  max: number
+  min: number
+  size: number
+}
+
+const PANE_SIDEBAR_SIZE = ref(25)
+const PANE_MAIN_SIZE = ref(75)
+const PANE_MAIN_TOP_SIZE = ref(45)
+const PANE_MAIN_BOTTOM_SIZE = ref(65)
+const PANE_STORAGE_KEY = `${props.layoutId}-app-pane-layout`
```

Comment:
Remove this line. We can access prop directly from methods. `layoutId` could be undefined. In that case no meaning of concatination.

Reply:
In that case, can set a condition if `layoutId` undefined no storage key will set?